third_party/3/orjson.pyi

```diff
@@ -0,0 +1,14 @@
+# https://github.com/ijl/orjson/blob/master/orjson.pyi
+
+from typing import Any, Callable, Optional, Union
+
+def dumps(
+    obj: Any, default: Optional[Callable[[Any], Any]], option: Optional[int]
+) -> bytes: ...
+def loads(obj: Union[bytes, str]) -> Union[dict, list, int, float, str, None]: ...
+
+class JSONDecodeError(ValueError): ...
```

Comment:
Since this is just a re-export of `JSONDecodeError` from `json` (if I understand the code correctly), I'd recommend to do the same thing in the stubs:

```python
from json import JSONDecodeError as JSONDecodeError
```

Reply:
This fails CI on python3.4. The package isn't compatible with python3.4, so in practice it's not an issue. I can make it conditional on version, but I don't know what's preferred here and have kept it as it was before.

```
running mypy --python-version 3.4 --strict-optional --no-site-packages --show-traceback --no-implicit-optional # with 781 files
third_party/3/orjson.pyi:4: error: Module 'json' has no attribute 'JSONDecodeError'
```

Reply:
Ok, I am fine with leaving it as is.